src/main/java/org/opensearch/flowframework/common/FlowFrameworkSettings.java

```diff
@@ -129,6 +133,25 @@ public class FlowFrameworkSettings {
         Setting.Property.NodeScope
     );
 
+    /** This setting sets max workflows that can be simultaneously provisioned, or reprovisioned by the same tenant */
+    public static final Setting<Integer> MAX_ACTIVE_PROVISIONS_PER_TENANT = Setting.intSetting(
+        "plugins.flow_framework.max_active_provisions_per_tenant",
+        2,
```

Comment:
Why are we keeping the default value as 2 here?

Reply:
I want the default value to be less than our thread pool size. 

What other default value do you suggest?